app/models/playback/http/GetTrust.scala

```diff
@@ -154,9 +154,9 @@ case class DisplayTrustLeadTrusteeOrgType(
                                            name: String,
                                            phoneNumber: String,
                                            email: Option[String] = None,
-                                           identification: DisplayTrustIdentificationOrgType,
+                                           identification: Option[DisplayTrustIdentificationOrgType],
```

Comment:
Identification is a required field in the response, what's the reason this was made optional?

Reply:
Reverted back to non optional.